gst-plugins-good.yaml

```diff
@@ -0,0 +1,115 @@
+package:
+  version: "1.27.2"
+  name: gst-plugins-good
+  epoch: 0
+  description: GStreamer streaming media framework good plug-ins
+  copyright:
+    - license: LGPL-2.1
+
+environment:
+  contents:
+    packages:
+      - alsa-lib-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cdparanoia-dev
+      - cmake
+      - expat-dev
+      - gdk-pixbuf
+      - glib-dev
+      - glib-gir
+      - gobject-introspection-dev
+      - graphene-dev
+      - gst-plugins-base-dev
+      - gstreamer
+      - gstreamer-dev
+      - gtk-3-dev
+      - libdrm-dev
+      - libice-dev
+      - libogg-dev
+      - libsm-dev
+      - libtheora-dev
+      - libvorbis-dev
+      - libxv-dev
+      - mesa-dev
+      - mesa-gbm
+      - mesa-glx
+      - mesa-libgallium
+      - meson
+      - opencore-amr
+      - opencore-amr-dev
+      - opus-dev
+      - orc-compiler
+      - orc-dev
+      - perl
+      - python3
+      - qt5-qtbase-dev
+      - qt5-qtbase-x11
+      - qt6-qtbase-dev
+      - qt6-qtbase-x11
+      - samba-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GStreamer/gstreamer
+      tag: ${{package.version}}
+      expected-commit: 285de3cf1de94b5b370a1af39446a7d1498d7c2c
+
+  - working-directory: subprojects/gst-plugins-good
+    pipeline:
+      - uses: meson/configure
+        with:
+          opts: |
+            -Db_lto=true \
+            -Dpackage-origin="https://github.com/wolfi-dev" \
+            -Dpackage-name="GStreamer good plug-ins (Wolfi-Linux)" \
+            -Dtests=disabled \
+            -Damrnb=enabled \
+            -Damrwbdec=enabled
+      - uses: meson/compile
+      - uses: meson/install
+      - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    pipeline:
+      - uses: split/dev
+    description: ${{package.name}} dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
+        - uses: test/tw/ldd-check
+          with:
+            packages: ${{subpkg.name}}
+
+  - name: ${{package.name}}-doc
+    pipeline:
+      - uses: split/manpages
+    description: ${{package.name}} manpages
+
+update:
+  enabled: true
+  git:
+    tag-filter-prefix: "1."
+
+test:
+  pipeline:
+    - uses: test/tw/ver-check
+      with:
+        bins: gst-inspect-1.0
```

Comment:
Given that gstreamer provides this binary I think checking the version of it is an unnecessary test.